pennylane/qinfo/entropies.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Differentiable quantum entropies"""
+
+import pennylane as qml
+
+
+def vn_entropy_transform(qnode, indices=None, base=None):
```

Comment:
Same as above. The user should always pass in the indices.
```suggestion
def vn_entropy_transform(qnode, indices, base=None):
```